accord-core/src/main/java/accord/utils/btree/UpdateFunction.java

```diff
@@ -74,10 +76,35 @@ Simple<V> flip()
         }
     }
 
-    static final Simple<Object> noOp = Simple.of((a, b) -> a);
+    final class NoOp<V> extends Simple<V>
```

Comment:
Backport to C*?

Reply:
I would prefer to say we will properly share the code at some point in the next 6-12 mo